File: frontend/src/app/domain/dto/statistics.ts
export class Statistics implements app.domain.dto.IStatistics, Serializable<Statistics> {
    appTotalSpend: number;
    weeklyTotalAmount: number;
    monthlyTotalAmount: number;
    weeklyHealthyOrders: number;
    yearlyTotalAmount: number;
    yearlyPastas: number;
    prepayedTotal: number;

    deserialize(input: any): Statistics {
        this.appTotalSpend = input.appTotalSpend;
        this.weeklyTotalAmount = input.weeklyTotalAmount;
        this.monthlyTotalAmount = input.monthlyTotalAmount;
        this.weeklyHealthyOrders = input.weeklyHealthyOrders;
        this.yearlyTotalAmount = input.yearlyTotalAmount;
        this.yearlyPastas = input.yearlyPastas;
        this.prepayedTotal = input.prepayedTotal;
        return this;
    }
}

File: backend/WebApi/Lunchorder.Domain/Dtos/Statistics.cs
namespace Lunchorder.Domain.Dtos
{
    public class Statistics
    {
        /// <summary>
        /// The total amount of money spent on the application
        /// </summary>
        public decimal AppTotalSpend { get; set; }

        /// <summary>
        /// The total amount spent for this week
        /// </summary>
        public decimal WeeklyTotalAmount { get; set; }

        /// <summary>
        /// The total amount spent for this month
        /// </summary>
        public decimal MonthlyTotalAmount { get; set; }

        /// <summary>
        /// The number of healthy orders this week
        /// </summary>
        public int WeeklyHealthyOrders { get; set; }

        /// <summary>
        /// The total amount spent for this year
        /// </summary>
        public decimal YearlyTotalAmount { get; set; }

        /// <summary>
        /// The number of pastas ordered for a year
        /// </summary>
        public int YearlyPastas { get; set; }

        /// <summary>
        /// The total amount of prepayed money
        /// </summary>
        public decimal PrepayedTotal { get; set; }
    }  
}